components/chart/plot-bands.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Plot Bands
+page_title: Plot Bands
+description: Plot Bands in the Charts for Blazor.
+slug: chart-plot-bands
+tags: telerik,blazor,chart,plot,bands,plotbands
+published: true
+position: 45
+---
+
+# Plot Bands
+
+Plot Bands are colored ranges in the Telerik UI for Blazor Chart, the purpose of which is to highlight areas of the chart by changing its background in a predefined axis range.
+
+The plot bands appear on the chart as colored areas that fill the entire space between two grid lines on the x-axis or on the y-axis.
```

Comment:
Perhaps we can merge this information with the previous paragraph? Currently it sounds a bit like a duplication.